src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -35,6 +35,12 @@ import ContentListFilters from './ContentListFilters';
 import Hide from '../Hide/Hide';
 import EmptyTableState from './components/EmptyTableState';
 import { useQueryClient } from 'react-query';
+import StatusIcon from './components/StatusIcon';
+
+export const red = global_danger_color_100.value;
+export const green = global_success_color_100.value;
```

Comment:
if you must use this in multiple components, we should likely create a "helper.ts" file for this.